curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/use-a-template-engines-powers.english.md

```diff
@@ -8,10 +8,10 @@ challengeType: 2
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-advancednode/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-advancednode/'>GitHub</a>.
 One of the greatest features of using a template engine is being able to pass variables from the server to the template file before rendering it to HTML.
-In your Pug file, you're about to use a variable by referencing the variable name as <code>#{variable_name}</code> inline with other text on an element or by using an equal side on the element without a space such as <code>p= variable_name</code> which sets that p elements text to equal the variable.
-We strongly recommend looking at the syntax and structure of Pug <a href='https://github.com/pugjs/pug'>here</a> on their Githubs README. Pug is all about using whitespace and tabs to show nested elements and cutting down on the amount of code needed to make a beautiful site.
-Looking at our pug file 'index.pug' included in your project, we used the variables <em>title</em> and <em>message</em>
-To pass those alone from our server, you will need to add an object as a second argument to your <em>res.render</em> with the variables and their value. For example, pass this object along setting the variables for your index view: <code>{title: 'Hello', message: 'Please login'</code>
+In your Pug file, you're able to use a variable by referencing the variable name as <code>#{variable_name}</code> inline with other text on an element or by using an equal sign on the element without a space such as <code>p=variable_name</code> which assigns the variable's value to the p element's text.
+We strongly recommend looking at the syntax and structure of Pug <a href='https://github.com/pugjs/pug'>here</a> on GitHub's README. Pug is all about using white space and tabs to show nested elements and cutting down on the amount of code needed to make a beautiful site.
```

Comment:
I don't think "whitespace" needs a space in  between  - https://en.wikipedia.org/wiki/Whitespace_character
```suggestion
We strongly recommend looking at the syntax and structure of Pug <a href='https://github.com/pugjs/pug'>here</a> on GitHub's README. Pug is all about using whitespace and tabs to show nested elements and cutting down on the amount of code needed to make a beautiful site.
```

Reply:
@Manish-Giri There is always another Wikipedia article which says something different.
https://en.wikipedia.org/wiki/White_space

I think it can go either way.

Reply:
The words `white space` are used to refer to a bunch of `whitespace characters`, so I think it's fine like this